docs/cloud/dev/spryker-cloud-commerce-os/managing-parameters-in-the-parameter-store.md

```diff
@@ -1,20 +0,0 @@
----
-title: Managing parameters in the Parameter Store
-description: Learn how to manage parameters in the Parameter Store.
-template: howto-guide-template
-originalLink: https://cloud.spryker.com/docs/managing-parameters-in-the-parameter-store
-originalArticleId: 2047c84c-bd7b-4bce-9203-08945367ad46
-redirect_from:
-  - /docs/managing-parameters-in-the-parameter-store
-  - /docs/en/managing-parameters-in-the-parameter-store
----
-
-This document describes how you can add new parameters and edit the values of existing parameters. You can use parameters for multiple purposes, like storing mail server details or providing Composer authentication details to the build and deploy process securely.
-
-## Adding new parameters
-
-Currently, only our team can add new parameters. To add a new parameter, fill out the [parameter creation form](https://spryker.force.com/support/s/hosting-change-requests/change-request-new-param-store-variable).
-
-## Changing parameters
```

Comment:
this part is still relevant. Except we will give them (customers) a "strict" access to SSM.